code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -446,14 +448,22 @@ This saves us from having to call add_fingerprint() any time something is put in
 			style_factor += r_ear.get_style()
 	if (glasses && !(head_coverage & COVERS_EYES))
 		style_factor += glasses.get_style()
+	else
+		style_factor += 1 // if we're not wearing any glasses we look stylish
 	if (gloves && !(suit_coverage & COVERS_FOREARMS))
 		style_factor += gloves.get_style()
+	else
+		style_factor -= 1 // if we're not hiding fingerprints we're definitely stylish
```

Comment:
This is a minus (-=), but text implies it should be a plus(+=).
I don't think it should be there, personally.

Reply:
I'll fix it, but why do you think not wearing gloves shouldn't be stylish?

Reply:
Because wearing gloves logically seems more stylish than not wearing em.
Then again encouraging people to go around revealing their fingerprints seems blaze so I'm neutral on it now.

Reply:
That vagabond skull sure looks squishy, would be a shame if a roach would bite it off...